modules/tpmnBidAdapter.js

```diff
@@ -64,7 +66,86 @@ export const spec = {
     // our server directly returns the format needed by prebid.js so no more
     // transformation is needed here.
     return bidResults;
-  }
+  },
+
+  getUserSyncs: function (syncOptions, serverResponses, gdprConsent, uspConsent) {
+    const syncArr = [];
+    if (syncOptions.iframeEnabled) {
+      let policyParam = '';
+      if (gdprConsent && gdprConsent.consentString) {
+        if (typeof gdprConsent.gdprApplies === 'boolean') {
+          policyParam += `&gdpr=${Number(gdprConsent.gdprApplies)}&gdpr_consent=${gdprConsent.consentString}`;
+        } else {
+          policyParam += `&gdpr=0&gdpr_consent=${gdprConsent.consentString}`;
+        }
+      }
+      if (uspConsent && uspConsent.consentString) {
+        policyParam += `&ccpa_consent=${uspConsent.consentString}`;
+      }
+      const coppa = config.getConfig('coppa') ? 1 : 0;
+      policyParam += `&coppa=${coppa}`;
+      syncArr.push({
+        type: 'iframe',
+        url: IFRAMESYNC + policyParam
+      })
+    } else {
+      const uuid = getUuid();
+      if (uuid) {
+        const bidswitchId = storage.getCookie('bidswitch');
```

Comment:
this will check for the presence of this cookie id in the first party, right? Seems it would always be true, even if bidswitch has a cookie.

Reply:
Yes, checking "first party cookie".

I want to use user sync with "bid-switch" based on tpmn.

Reply:
I think it would be better not to uuid for user sync, so I modified the code.